build_scripts/build_runner.py

```diff
@@ -392,7 +401,8 @@ def generate_build_config(self):
             'vs_component': self._vs_component,
             'DEFAULT_OPTIONS': self.options,
             'Stage': Stage,
-            'copy_win_files': copy_win_files
+            'copy_win_files': copy_win_files,
+            'args': self.config_args
```

Comment:
add TODO: unify naming convention of the API. There 3 different now, e.g. DEFAULT_OPTIONS, Stage, args